examples/predictors/lstm/tests/test_xprize_predictor.py

```diff
@@ -10,15 +10,10 @@
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 FIXTURES_PATH = os.path.join(ROOT_DIR, 'fixtures')
-EXAMPLE_INPUT_FILE = os.path.join(ROOT_DIR, "../../../validation/data/2020-08-01_2020-08-04_ip.csv")
+EXAMPLE_INPUT_FILE = os.path.join(ROOT_DIR, "../../../../validation/data/2020-09-30_historical_ip.csv")
 DATA_FILE = os.path.join(FIXTURES_PATH, "OxCGRT_latest.csv")
 DATA_URL = "https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv"
-PREDICTOR_27 = os.path.join(FIXTURES_PATH, "pred27", "predictor.h5")
```

Comment:
These old models are not 'compatible' anymore. And not used in this test anymore.